kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/WebUtils.java

```diff
@@ -0,0 +1,17 @@
+package com.provectus.kafka.ui.extensions;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+
+public class WebUtils {
+
+    public static boolean isVisible(SelenideElement element) {
+        boolean isVisible = false;
+        try {
+            isVisible = element.is(Condition.visible);
```

Comment:
let's use smth like this here:
element.shouldBe(Condition.visible);
isVisible = true;

Reply:
Fixed